eng/common/scripts/common.ps1

```diff
@@ -60,3 +60,4 @@ $GetRepositoryLinkFn = "Get-${Language}-RepositoryLink"
 $GetEmitterAdditionalOptionsFn = "Get-${Language}-EmitterAdditionalOptions"
 $GetEmitterNameFn = "Get-${Language}-EmitterName"
 $GetEmitterPackageJsonPathFn = "Get-${Language}-EmitterPackageJsonPath"
+$GetEmitterPackageLockPathFn = "Get-${Language}-EmitterPackageLockPath"
```

Comment:
Do we need another function? Any reason we cannot assume it is in the same path as the packageJson?

Reply:
This is a good point.  I think it's fine to require lockfile in same folder.

Reply:
The path is a full path.   The override Get-${Language}-EmitterPackageJsonPath allows the author to point to any path. It would be odd to assume that there will be `emitter/emitter-package-lock.json` beside `emitter/package.json`

Reply:
If we do stick with both of these can we have a common convention, we can implement here instead of requiring them to be implemented in all languages. They can still be overridden if we do have cases that are special.

Reply:
@hallipr: Can you share an example of where an SDK dev would want to put the lockfile in a different folder?

Reply:
No, but in the same way that I can't share an example of where an SDK dev would want to put the emitter-package.json in a different location.  Get-${Language}-EmitterPackageJsonPath is currently unused in any repo.

Rather than let the someone explicitly override `emitter-package.json` but leave `emitter-package-lock.json` inferred from that path, I'd rather we have both or neither.

Reply:
> If we do stick with both of these can we have a common convention, we can implement here instead of requiring them to be implemented in all languages. They can still be overridden if we do have cases that are special.

There is a common convention of `eng/emitter-package(-lock).json` with the override if required.  No one currently overrides either.

Reply:
OK that is good, I didn't realize no one was overriding this yet. If that is the case I have to wonder if we should just remove these until a point where we need someone to override.